flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -389,6 +393,27 @@ public String cancelWithSavepoint(JobID jobId, @Nullable String savepointDirecto
 		});
 	}
 
+	@Override
+	public Map<String, Object> getAccumulators(final JobID jobID) throws Exception {
```

Comment:
we should also override `Map<String, Object> getAccumulators(JobID jobID, ClassLoader loader)`